go.mod

```diff
@@ -1,25 +1,25 @@
 module github.com/cilium/proxy
 
-go 1.20
+go 1.21
```

Comment:
we might want to set the version with the patch version

Reply:
ah good point :+1: